tests/bootstrap.php

```diff
@@ -5,3 +5,11 @@
     $loader = require __DIR__ . '/../../../../vendor/autoload.php';
 }
 $loader->addPsr4('React\\Tests\\EventLoop\\', __DIR__);
+
+if (!defined('SIGUSR1')) {
+    define('SIGUSR1', 1);
+}
+
+if (!defined('SIGUSR2')) {
+    define('SIGUSR2', 2);
+}
```

Comment:
How about making a package both Amp and ReactPHP can use that polyfills the signal constants?

Reply:
For two constants not really. Seems way to much overhead tbh.

Reply:
It's not only those two. It's also for using them with other reactor implementations if `pcntl` isn't available.

Reply:
@kelunik I agree with @WyriHaximus, this is probably overkill for this test suite. But I agree that it may make sense for consumers that want to rely on signal constant names, so I'd suggest go for it and create that package :shipit: